CPE200Lab1/CPE200Lab1/CalculatorEngine.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CPE200Lab1
+{
+    public class CalculatorEngine
+    {
+        public string calculate(string operate, string firstOperand, string secondOperand, int maxOutputSize = 8)
+        {
+            double result;
+            string[] parts;
+            int remainLength;
+            switch (operate)
+            {
+                case "+":
+                    return (Convert.ToDouble(firstOperand) + Convert.ToDouble(secondOperand)).ToString();
+                case "-":
+                    return (Convert.ToDouble(firstOperand) - Convert.ToDouble(secondOperand)).ToString();
+                case "X":
+                    return (Convert.ToDouble(firstOperand) * Convert.ToDouble(secondOperand)).ToString();
+                case "÷":
+                    // Not allow devide be zero
+                    if (secondOperand != "0")
```

Comment:
What if secondOperand is zero, then your code will do nothing, right? so the user will confuse why it does nothing in response to = key. Better show error